src/assets/scss/application.scss

```diff
@@ -239,6 +239,29 @@ $nav-active-border-thickness: 5px;
   }
 }
 
+.account-header-navigation__button {
+  @include govuk-font($size: 19, $weight: bold);
+  border: 0 none;
+  cursor: pointer;
```

Comment:
This is the same selector and rules as L249 (minus font size). Since L249 and below takes precedence, can we remove this?

Reply:
Of course, I don't know how I missed this, probably end of the year brain fog